docs/user/00-02-technical-reference.md

```diff
@@ -0,0 +1,17 @@
+# Technical reference
+
+In the following documents, you'll find the architecture documents, the configuration parameters, custom resources (CRs) and any other useful references:
+
+- [Serverless architecture](/docs/user/04-10-architecture.md)
+- [Internal Docker registry](/docs/user/04-20-internal-registry.md)
+- [Serverless chart](/docs/user/05-10-chart.md)
```

Comment:
```suggestion
```
Chart is no longer relevant for the user ( actually never was).
Serverless CR becomes the API to configure serverless installation